src/components/xray/XrayExportSession.vue

```diff
@@ -4,7 +4,7 @@
       <v-avatar size="24">
```

Comment:
@Zubs - I had to also update this to: `<v-avatar width="12" size="24">` for it to fix the distortion

Reply:
Fixed